routes/orders.js

```diff
@@ -14,9 +14,10 @@ router.route('/:id')
 
 router.route('/payment')
     .post(verification.verifyCart(), ordersController.checkoutNewOrder)
-
+//verification.verifyCart(), verification.verifyPaymentForm(),
+// remove verfication caused error on redirect to this path
 router.route('/payment/summary')
```

Comment:
y ? 
Maybe you entered not a real email ?
I think we should debug this and return the `verification.verifyCart() and verification.verifyPaymentForm()`